bibtex/bib/quantum.bib

```diff
@@ -1911,6 +2013,22 @@ @Article{SAA+10
   Url                      = {http://dx.doi.org/10.1140/epjd/e2010-00103-y}
 }
 
+@Article{SMN10,
+  Title                    = {The Case for Quantum Key Distribution},
+  Author                   = {Stebila, Douglas and Mosca, Michele and LÃ¼tkenhaus, Norbert},
+  Journal                  = {Quantum Communication and Quantum Networking},
+  Year                     = {2010},
+  Pages                    = {283â€“296},
```

Comment:
yeah, that comes from JabRef's automatic fetch from the DOI. It also got the type of publication wrong---this was actually a proceedings paper---which I also fixed.